File: java/src/main/java/com/adaptionsoft/games/uglytrivia/domain/events/DiceRolled.java
package com.adaptionsoft.games.uglytrivia.domain.events;

public class DiceRolled {
    public final String playerName;
    public final int roll;

    public DiceRolled(String playerName, int roll) {
        this.playerName = playerName;
        this.roll = roll;
    }
}

File: php/src/Domain/Event/DiceRolled.php
<?php


namespace Trivia\Domain\Event;


class DiceRolled
{
    /**
     * @var string
     */
    public $playerName;
    /**
     * @var int
     */
    public $roll;

    /**
     * DiceRolled constructor.
     * @param string $playerName
     * @param $roll
     */
    public function __construct(string $playerName, int $roll)
    {
        $this->playerName = $playerName;
        $this->roll = $roll;
    }
}